### PR TITLE
refactor: wallet selection on the transfer screen

### PR DESCRIPTION
**Pull request by luciorubeens**
## Summary

- Remove ionic4-auto-complete package
- Add wallet picker component and modal

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: src/app/components/close-popup/close-popup.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<ion-button fill="clear" (tap)="onClose()" [color]="color">
+<ion-button fill="clear" (click)="onClose()" [color]="color">
```
Comment (clucasalcantara) [inline src/app/components/close-popup/close-popup.html]: Awesome work, man :3

Just a doubt here, why we are changing from `tap` to `click`? The click makes the action fired immediately but could bring some problems on mobile, isn't better to switch to `press` instead?

File: src/app/components/wallet-picker/wallet-picker.component.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<ion-list>
+	<ion-list-header class="font-bold capitalize pl-0" mode="md">
+		{{ title | translate }}
+	</ion-list-header>
+	<ion-item button (click)="onTap(wallet)" *ngFor="let wallet of wallets">
```
Comment (clucasalcantara) [inline src/app/components/wallet-picker/wallet-picker.component.html]: Same here, we shouldn't be using `press`?

File: src/app/components/close-popup/close-popup.html (hunk 1 above)
Comment (luciorubeens) [reply src/app/components/close-popup/close-popup.html]: First review :noah:

This tap stuff is an old fix when Ionic 2 did not handle it properly, so it was handled by hammer.js. But it has now been fixed and this close button is not working atm.

File: src/app/components/wallet-picker/wallet-picker.component.html (hunk 2 above)
Comment (luciorubeens) [reply src/app/components/wallet-picker/wallet-picker.component.html]: (`press`) means long press: https://hammerjs.github.io/recognizer-press/

**Pull request merged**